src/install.ts

```diff
@@ -55,7 +51,7 @@ export function install(Vue: VueConstructor) {
   }
 
   if (__DEV__) {
-    if (Vue.version[0] !== '2' || Vue.version[1] !== '.') {
+    if (Vue.version && (Vue.version[0] !== '2' || Vue.version[1] !== '.')) {
```

Comment:
This is nice to have, however, should we throw another warning when Vue.version doesn't exist?

Reply:
When you do `Vue.extend` it doesn't come with the version :/ 

Currently it throws an exception because version is `undefined`

Reply:
ok. Should we validate whether the `Vue` passed is a valid Vue constructor or we just let it go? Is there any way to do that?

Reply:
Probably better just warn if `Vue.version` is not present, what do you think?

Reply:
Good to me :)